src/Uno.Toolkit.UI/Behaviors/ProgressExtensions.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using Windows.System;
+using Microsoft.Extensions.Logging;
+using Uno.Extensions;
+using Uno.Logging;
+using System.Diagnostics.CodeAnalysis;
+using Uno.UI.Extensions;
+using Windows.UI.Xaml.Media;
+
+
+
+#if IS_WINUI
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Input;
+#else
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Input;
+using ItemsRepeater = Microsoft.UI.Xaml.Controls.ItemsRepeater;
+#endif
+
+
+namespace Uno.Toolkit.UI
+{
+	public static class ProgressExtensions
+	{
+		public static readonly DependencyProperty IsActiveProperty =
```

Comment:
it is better for you to copy the template like here: 
https://github.com/unoplatform/uno.toolkit.ui/blob/main/src/Uno.Toolkit.UI/Behaviors/DrawerControlBehavior.cs#L35-L49

without setter/getter inside the #region
and the handler outside

Reply:
could also use a short `<summary>` to describe what it does:
> recursively activate/disactivate all nested progress ring/bar